### PR TITLE
add foresting setup for data and MC

### PR DESCRIPTION
**Pull request by ginnocen**

### CODE REVIEW

**Comment by ginnocen** [comment]
@Cristian-Baldenegro

File: ForestSubmission/20241121_ForestSubmissionGtoCCbar2018/forest_miniAOD_run2_DATA.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+
+process.Trigger = cms.EDFilter( "TriggerResultsFilter",
+      triggerConditions = cms.vstring(
+         #'HLT_HIEle*',
+         #'HLT_HIDoubleEle*',
+         'HLT_HIL3Mu12_*',
+         'HLT_HIL3Mu15_*',
+         'HLT_HIL3Mu20_*',
+         'HLT_HIL3DoubleMu*',
```
Comment (ginnocen) [inline ForestSubmission/20241121_ForestSubmissionGtoCCbar2018/forest_miniAOD_run2_DATA.py:140]: @Cristian-Baldenegro could you check with Florian to see which one of these three HLT paths we should be using?

Comment (Cristian-Baldenegro) [reply ForestSubmission/20241121_ForestSubmissionGtoCCbar2018/forest_miniAOD_run2_DATA.py:140]: Checked with Florian, we can use `HLT_HIL3DoubleMuOpen` for this analysis.

Comment (ginnocen) [reply ForestSubmission/20241121_ForestSubmissionGtoCCbar2018/forest_miniAOD_run2_DATA.py:140]: OK, does he know what are HLT_HIL1DoubleMuOpen and HLT_HIL2DoubleMuOpen?

**Pull request merged**